src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/model/Email.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.model;
+
+import java.util.Map;
+
+public class Email {
```

Comment:
Could we call this "EmailDocument" it will be consistent with what we have in other apps.

Reply:
if this represents an email I'm happy with email

Reply:
maybe get martin to decide!

Reply:
It represents the data that we put as a message on a kafka topic that the email sender constructs an email from. 
Not sure how best to describe it but I don't really know what "Email Document" means

Reply:
EmailContent?

Reply:
It's just less confusing if someone looks at extension and ptf and wishes to see the equivalent in objections.

Reply:
we should try and improve with each project otherwise we'd be stuck with how it was done in extensions forever

Reply:
In that case we should probably have something more descriptive of what it is for, something like "EmailWrapper" or "EmailContent".

Reply:
EmailContent it is